src/graphql/schema.gql

```diff
@@ -385,6 +391,7 @@ type Space {
   verified: Boolean
   flagged: Boolean
   rank: Float
+  created_at: Int!
```

Comment:
Maybe we have rename this field to `created` and `updated_at` like all other tables first???

Reply:
This change needs some more planification, as it may breaks the UI and other services using that field, unless we can alias both field in a transition period

Reply:
> as it may breaks the UI and other services using that field

How other services or UI using this field when this PR is the first time we adding this field 😅

Reply:
Valid point

Reply:
In that case, we should also rename the `updated_at` field in Space, for consistency.

In another note, the `_at` suffix for timestamp property make more sense, as it differentiate if from boolean field (`verified`), and it's also the convention used by some CMS, like Ruby on Rails and Symfony.

Reply:
done